packages/design-system-scripts/gulp/docs/generatePages/generatePages.js

```diff
@@ -200,32 +200,51 @@ module.exports = async function generatePages(sourceDir, docsDir, options, chang
       processKssSection(kssSection, options)
     )
   );
+  // Get page sections that come from core and child design systems
```

Comment:
`pageSections` is just all the doc site sections from KSS and markdown files, which may or may not be from the core or child DS. 

I think a better description here would be:
"Combine KSS and Markdown page sections and add cmsds links to for child design systems"